.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- [ ] The linter checks are not detected. Ensure that they pass and a green tick button is displayed.